### DISCUSSION TITLE
Images are being cropped

### DISCUSSION THREAD

**Discussion by RiverrApp** [soimy/maxrects-packer#57] [Q&A]
Hello! I am a complete noob so I apologize for asking a simple question. I am successfully "packing" my images, but it is cropping my images when there isnt enough space for an entire image to fit. Can you show me where in the documentation I can learn about this and fix it so that it will not add an image if it is not large enough to fit in the bin?

Thanks!

**Comment by soimy** [comment]
Sure, can you provide an example so I can represent the issue "Images being cropped"?
As designed, the images that cannot fit into the existing bin shall be moved to the new bin.
